Sources/Apodini/Properties/ObservedObject.swift

```diff
@@ -7,17 +7,16 @@ import Foundation
 /// This is helpful for service-side streams or bidirectional communication.
 @propertyWrapper
 public struct ObservedObject<Element: ObservableObject>: Property {
-    private var objectIdentifer: ObjectIdentifier?
+    private var objectIdentifier: ObjectIdentifier?
     private var element: Element?
-    private var changedWrapper: Wrapper<Bool>
-    private var wrappedValueDidChange: Wrapper<(() -> Void)?>
+    private let _initializer: (() -> Element)?
     
     public var wrappedValue: Element {
         get {
             if let element = element {
                 return element
             }
-            if let objectIdentifer = objectIdentifer,
+            if let objectIdentifer = objectIdentifier,
                let element = EnvironmentValues.shared.values[objectIdentifer] as? Element {
                 return element
             }
```

Comment:
Does the bug with the missing `ObjectIdentifier` in in the environment still occur or is there an update?

Reply:
It is gone. We actually forgot to add the according element to the environment🙈. It seemed so buggy because it was added asynchronously afterwards and thus was present at the time one would start debugging.